addons/web/static/src/core/signature/name_and_signature.scss

```diff
@@ -6,7 +6,7 @@
 .o_signature_stroke {
     position: absolute;
     border-top: #D1D0CE solid 2px;
-    bottom: 33%;
+    bottom: 20%;
```

Comment:
Why this change ? Is it related to the `devicePixelRatio` ?

Reply:
Same, as we move the signature with the yOffset, this change move the line too to avoid crossing out the signature. I will move these changes into a separate commit.